worker/src/core/services/notification.service.ts

```diff
@@ -0,0 +1,105 @@
+import MailClient, {
+  MailToSend,
+  SendEmailOpts,
+} from '@shared/clients/mail-client.class'
+import config from '@core/config'
+import { loggerWithLabel } from '@core/logger'
+import { ThemeClient } from '@shared/theme'
+import { TemplateClient } from '@shared/templating'
+
+const logger = loggerWithLabel(module)
+
+const sendEmail = async (
+  mailClient: MailClient,
+  mail: MailToSend,
+  opts?: SendEmailOpts
+): Promise<string | void> => {
+  try {
+    return mailClient.sendMail(mail, opts)
+  } catch (e) {
+    logger.error({
+      message: 'Error while sending test email',
```

Comment:
I think "test email" is a bit of a misnomer?

Reply:
fixed w next push